packages/promotion/src/services/promotion-module.ts

```diff
@@ -794,7 +823,7 @@ export default class PromotionModuleService<
   @InjectTransactionManager("baseRepository_")
   protected async createPromotionRulesAndValues_(
     rulesData: PromotionTypes.CreatePromotionRuleDTO[],
-    relationName: "promotions" | "application_methods",
+    relationName: "promotions" | "method_target_rules" | "method_buy_rules",
```

Comment:
suggestion: Goes with my other comment
```suggestion
    relationName: "promotions" | "target_rules" | "buy_rules",
```